packaging/scripts/install.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+
+install_debian_like() {
+  # This is needed to ensure package installs don't prompt for any user input.
+  export DEBIAN_FRONTEND=noninteractive
+
+  apt-get update
+
+  # Install NetData
+  apt-get install -y "/artifacts/netdata_${VERSION}_${ARCH}.deb"
+
+  # Install testing tools
+  apt-get install -y --no-install-recommends \
+    curl netcat jq
+}
+
+install_fedora_like() {
+  # Using a glob pattern here because I can't reliably determine what the
+  # resulting package name will be (TODO: There must be a better way!)
+
+  # Install NetData
+  dnf install -y /artifacts/netdata-"${VERSION}"-*.rpm
+
+  # Install testing tools
+  dnf install -y curl nc jq
```

Comment:
Just noticed this, but this will fail on CentOS 7 (it still uses `yum`).

Reply:
Fixed

Reply:
Just writing a quick `./packaging/build_package_install_test.sh` script as a quick starting point for kickstarting the new CI/CD for all of this.

I got sick and tired of typing :)

Reply:
Will mark this as resolved once I have the script working and CentOS 7 tested.